$NetBSD$

- Allow setting PREFIX
- manpages should be installed to $(PREFIX)/man

Not yet upstreamed as other changes are likely needed.
See TODO.

--- Makefile.orig	2018-07-27 08:10:35.000000000 +0000
+++ Makefile
@@ -4,7 +4,7 @@ CFLAGS += -std=c99 -Wall -Wextra -pedant
 TARGETS = f3write f3read
 EXTRA_TARGETS = f3probe f3brew f3fix
 
-PREFIX = /usr/local
+PREFIX?= /usr/local
 INSTALL = install
 LN = ln
 
@@ -23,9 +23,9 @@ extra: $(EXTRA_TARGETS)
 install: all
 	$(INSTALL) -d $(DESTDIR)$(PREFIX)/bin
 	$(INSTALL) -m755 $(TARGETS) $(DESTDIR)$(PREFIX)/bin
-	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/man/man1
-	$(INSTALL) -m644 f3read.1 $(DESTDIR)$(PREFIX)/share/man/man1
-	$(LN) -sf f3read.1 $(DESTDIR)$(PREFIX)/share/man/man1/f3write.1
+	$(INSTALL) -d $(DESTDIR)$(PREFIX)/man/man1
+	$(INSTALL) -m644 f3read.1 $(DESTDIR)$(PREFIX)/man/man1
+	$(LN) -sf f3read.1 $(DESTDIR)$(PREFIX)/man/man1/f3write.1
 
 install-extra: extra
 	$(INSTALL) -d $(DESTDIR)$(PREFIX)/bin
